roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -748,6 +748,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
+    ''' Find and return oc binary file '''
+    # https://github.com/openshift/openshift-ansible/issues/3410
+    # oc can be in /usr/local/bin in some cases, but that may not
+    # be in $PATH due to ansible/sudo
+    paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
```

Comment:
I'd like to see the `['/usr/local/bin', os.path.expanduser('~/bin')]` defined in a variable so we can reference it, update it, etc.  I could see us using that variable in another way at some point in the future.